README.md

```diff
@@ -23,13 +25,13 @@ Given this, we don’t recommend using the the GDK for Unreal for games in the p
 **Who is it for?**<br>
 We recommend you start using the GDK if:
 
-* You don’t mind the rough edges and are interested in contributing to the GDK (see our [contributions]({{urlRoot}}/contributing) policy).
+* You don’t mind the rough edges and are interested in contributing to the GDK (see our [contributions](https://docs.improbable.io/unreal/latest/contributing) policy).
```

Comment:
It's always struck me as odd that we say - you can contribute but...oh, you can't. :-)

Reply:
Do you think it's worth adding "on the documentation website"?

That is: "...(see our [contributions](https://docs.improbable.io/unreal/latest/contributing) policy on the documentation website).

Reply:
Discussed offline, we'll leave these for now.